type_lens/type_view.py

```diff
@@ -121,11 +121,16 @@ def is_subclass_of(self, cl: type[Any] | tuple[type[Any], ...]) -> bool:
             if self.origin in UNION_TYPES:
                 return all(t.is_subclass_of(cl) for t in self.inner_types)
 
-            return self.origin not in UNION_TYPES and issubclass(self.origin, cl)
+            return self.origin not in UNION_TYPES and isinstance(self.origin, type) and issubclass(self.origin, cl)
```

Comment:
Is this relevant to the PR, or accidental?

Reply:
bah! sorry yes. Messed up my rebase for https://github.com/litestar-org/type-lens/pull/9